README.Rmd

```diff
@@ -0,0 +1,135 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+
+# set a seed so that the waiting list example is consistent
+set.seed(2)
+```
+
+# NHSRwaitinglist
+
+<a alt="NHS-R Community's logo" href='https://nhsrcommunity.com/'><img src='https://nhs-r-community.github.io/assets/logo/nhsr-logo.png' align="right" height="80" /></a>
+
+<!-- badges: start -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-7-orange.svg?style=flat-square)](#contributors-)
+[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
+[![R-CMD-check](https://github.com/nhs-r-community/NHSRwaitinglist/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/nhs-r-community/NHSRwaitinglist/actions/workflows/R-CMD-check.yaml)
+[![Codecov test coverage](https://codecov.io/gh/nhs-r-community/NHSRwaitinglist/branch/main/graph/badge.svg)](https://app.codecov.io/gh/nhs-r-community/NHSRwaitinglist?branch=main)
+<!-- badges: end -->
+
+{NHSRwaitinglist} is an R package implementing the waiting list management approach described in the paper [Understanding Waiting Lists Pressures](https://www.medrxiv.org/content/10.1101/2022.08.23.22279117v1) by Fong et al.  The methodology is presented by Neil Walton, Professor in Operations Management at Durham University Business School.  The package is being contributed to by members of the [NHS-R Community](https://nhsrcommunity.com/), with the aim of making it simpler for NHS analysts to adopt these methods for the analysis of their own waiting lists.  
+
+## Explanatory Videos
+
+These videos explain more about the Operations Management methodologies which are encapsulated by the package.  Click through to view them on YouTube.
+
+Detailed presentation of method | Recap of method, and presentation of this package
+:---:|:---:
+[![Thumbnail for link to MAN huddle video](man/figures/NHSRwaitinglist_MAN_huddle_thumbnail.jpg)](https://www.youtube.com/watch?v=NWthhW5Fgls) | [![Thumbnail for link to Health and Care Analytics Conference E-lab video](man/figures/NHSRwaitinglist_HACA_elab_thumbnail.jpg)](https://www.youtube.com/watch?v=3peqTEl_ZAQ)
+Midlands Analyst Network Huddle, Nov 2023 | Health and Care Analytics Conference e-lab, Jul 2024
+
+## Installation 
+
+You can install the current version of {NHSRwaitinglist} from GitHub with:
```

Comment:
Could add a reference here to needing a GitHub PAT (e.g. [{usethis} guide](https://usethis.r-lib.org/articles/git-credentials.html#get-a-personal-access-token-pat) or [Happy Git With R](https://happygitwithr.com/https-pat.html?q=personal%20access#https-pat)).